recipes-debian/swig/swig.inc

```diff
@@ -0,0 +1,60 @@
+SUMMARY = "SWIG - Simplified Wrapper and Interface Generator"
```

Comment:
um, the difference from poky is remove SRC_URI.

```
-SRC_URI = "${SOURCEFORGE_MIRROR}/${BPN}/${BPN}-${PV}.tar.gz"
-
```
How about override SRC_URI after including meta/recipes-devtools/swig/swig.inc then includes recipes-debian/sources/swig.inc. [util-linux](https://github.com/miraclelinux/meta-debian-extended/blob/warrior/recipes-debian/util-linux/util-linux_debian.bb) does same thing. So we don't have to have ig/swig.inc locally.